mimic/canned_responses/maas_test_check.py

```diff
@@ -0,0 +1,67 @@
+"""
+Canned response for /test-check
+"""
+
+import collections
+
+
+def test_check(checkType):
+    """
+    Support test-check API
```

Comment:
This documentation isn't sufficient.  What does it mean to support the "test-check API"?  What is the parameter type? What is the return type?  How will a future maintainer understand this in context?